setup.py

```diff
@@ -9,12 +9,11 @@
 url = "https://github.com/ai2es/" + project_name
 # Classifiers listed at https://pypi.org/classifiers/
 classifiers = ["Programming Language :: Python :: 3"]
-setup(name="evidential", # Change
+setup(name="evidential",  # Change
       version=version,
       description=package_description,
       url=url,
       author="John Schreck",
```

Comment:
Can you update the author list (probably the four of us for now, maybe other people will contribute later)

Reply:
Edited.